### PR TITLE
+ht2 added rendering to PriorityFrame model and test case (#245)

### PR DESCRIPTION
**Pull request by gosubpl**
plus minor cleanup

### CODE REVIEW

**Comment by gosubpl** [comment]
Refs #245

**Comment by gosubpl** [comment]
Also refs #247

File: akka-http2-support/src/main/scala/akka/http/impl/engine/http2/FrameEvent.scala [hunk 1]
```diff
@@ -28,7 +28,6 @@ final case class HeadersFrame(
   endHeaders:          Boolean,
   headerBlockFragment: ByteString) extends StreamFrameEvent
 
-//final case class PriorityFrame(streamId: Int, streamDependency: Int, weight: Int) extends StreamFrameEvent
```
Comment (ktoso) [inline akka-http2-support/src/main/scala/akka/http/impl/engine/http2/FrameEvent.scala]: thanks, since it's already below.

**Comment by gosubpl** [comment]
Added initial entry point for handling `PriorityFrame` in the `Http2ServerDemux`

**Comment by gosubpl** [comment]
This is now imho ready for merge. I can continue work on PriorityFrame parsing after Demux revamp promised by @jrudolph takes place.

**Comment by jrudolph** [comment]
Thanks, @gosubpl. LGTM

**Comment by gosubpl** [comment]
@jrudolph @ktoso thx for reviewing!

**Pull request merged**